src/client/pfs/pfs.proto

```diff
@@ -561,6 +561,16 @@ message DiffFileResponseV2 {
   FileInfo new_file = 2;
 }
 
+message CreateTempFileSetResponse {
+  string fileset_id = 1;
+  int64 expires_at = 2;
+}
+
+message RenewTempFileSetRequest {
+  string fileset_id = 1;
+  int64 extend_seconds = 2;
```

Comment:
I think you may have mentioned this when we discussed this, but why is this not just interpreted as a duration as compared to number of seconds?

Reply:
Do you mean like using the well-known-type for duration? Depending on how protoc is setup those can be a pain, sometimes requiring another conversion in the code.  We don't really need sub-second resolution for leases, since they will typically be on the order of minutes or at lease tens of seconds.  If we use the duration types elsewhere, I can switch to those.